### PR TITLE
Lvnod 25 routers on react

### PR DESCRIPTION
**Pull request by denisbodnar**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-25)
* [JIRA subtask](https://ssu-jira.softserveinc.com/browse/LVNOD-38)

## Code reviewers

- [ ] @AlbaMontis
- [ ] @avoytovich
- [ ] @yuliannakorol

### Second Level Review

- [ ] @lhalam
- [ ] @4asof4ik

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo
## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=90%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: app/src/components/About/About.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default class About extends React.Component {
+  render() {
+    return (<h1>We are Da-Mi</h1>);
```
Comment (AlbaMontis) [inline app/src/components/About/About.js]: You can skip () here.

File: app/src/components/Address/Address.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default class Address extends React.Component {
+  render() {
+    return (<h2>You can find us in Lviv</h2>);
```
Comment (AlbaMontis) [inline app/src/components/Address/Address.js]: You can skip () here.

File: app/src/components/Home/Home.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default class Home extends React.Component {
+  render() {
+    return (<h1>Hello here, its alive!</h1>);
```
Comment (AlbaMontis) [inline app/src/components/Home/Home.js]: You can skip () here.

**Review by avoytovich — DISMISSED** [review]

**Review by avoytovich — DISMISSED** [review]

**Review by yuliannakorol — DISMISSED** [review]

**Review by AlbaMontis — DISMISSED** [review]

**Review by avoytovich — DISMISSED** [review]

**Review by yuliannakorol — DISMISSED** [review]

**Review by AlbaMontis — DISMISSED** [review]

**Review by AlbaMontis — DISMISSED** [review]

**Review by yuliannakorol — DISMISSED** [review]

**Review by avoytovich — DISMISSED** [review]

File: app/src/app.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { Hello } from './components';
+import { App } from './container';
```
Comment (4asof4ik) [inline app/src/app.js]: rename container folder to containers

File: app/src/components/Contacts/Contacts.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Link } from 'react-router';
```
Comment (4asof4ik) [inline app/src/components/Contacts/Contacts.js]: import but not used

**Review by AlbaMontis — DISMISSED** [review]

File: app/src/containers/App/App.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import Routes from '../../router/router';
+import { Router, browserHistory } from 'react-router';
+
+const Routing = <Router history={browserHistory} routes={Routes}/>;
```
Comment (4asof4ik) [inline app/src/containers/App/App.js]: if we write export default ...... in one line it should be in all project

File: app/src/router/router.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+const Routes = (
+  <Route path='/' component={Main}>
+    <IndexRoute component={Home}/>
+    <Route path='about' component={About}/>
+    <Route path='contacts' component={Contacts}/>
+  </Route>
+);
+
+export default Routes;
```
Comment (4asof4ik) [inline app/src/router/router.js]: fix here too

**Review by avoytovich — DISMISSED** [review]

**Review by AlbaMontis — APPROVED** [review]

**Pull request merged**